README.md

```diff
@@ -399,7 +399,7 @@ See [KVS WebRTC Access Control Documentation](https://docs.aws.amazon.com/kinesi
 #### Running WebRTC Test Page Locally
 The SDK and test page can be edited and run locally by following these instructions:
 
-NodeJS version 8+ is required.
+See the supported node versions in [ci.yml](.github/workflows/ci.yml).
```

Comment:
Isn't it better to specify the versions directly in the readme?

Reply:
Done